libraries/AP_HAL_F4Light/boards/f4light_MatekF405_OSD/system_stm32f4xx.c

```diff
@@ -333,7 +333,7 @@ void SystemCoreClockUpdate(void)
   SystemCoreClock >>= tmp;
 }
 
-extern void __error(uint32_t num, uint32_t pc, uint32_t lr);
+extern void __error(uint32_t num, uint32_t pc, uint32_t lr, uint32_t flag, uint32_t flag);
```

Comment:
copy, paste twice ?

Reply:
sure! The strange thing - there is no even GCC warning...